packages/contracts-bedrock/scripts/ChainAssertions.sol

```diff
@@ -20,38 +20,49 @@ import { ISystemConfigV0 } from "scripts/interfaces/ISystemConfigV0.sol";
 import { console2 as console } from "forge-std/console2.sol";
 
 library ChainAssertions {
-    /// @notice Asserts the correctness of an L1 deployment
+    /// @notice Asserts the correctness of an L1 deployment. This function expects that all contracts
+    ///         within the `prox` ContractSet are proxies that have been setup and initialized.
     function postDeployAssertions(
-        Types.ContractSet memory prox,
-        DeployConfig cfg,
-        uint256 l2OutputOracleStartingTimestamp,
-        Vm vm
+        Types.ContractSet memory _prox,
+        DeployConfig _cfg,
+        uint256 _l2OutputOracleStartingBlockNumber,
+        uint256 _l2OutputOracleStartingTimestamp,
+        Vm _vm
     )
         internal
         view
     {
-        ResourceMetering.ResourceConfig memory rcfg = SystemConfig(prox.SystemConfig).resourceConfig();
+        ResourceMetering.ResourceConfig memory rcfg = SystemConfig(_prox.SystemConfig).resourceConfig();
         ResourceMetering.ResourceConfig memory dflt = Constants.DEFAULT_RESOURCE_CONFIG();
         require(keccak256(abi.encode(rcfg)) == keccak256(abi.encode(dflt)));
 
-        checkSystemConfig(prox, cfg);
-        checkL1CrossDomainMessenger(prox, vm);
-        checkL1StandardBridge(prox);
-        checkL2OutputOracle(prox, cfg, l2OutputOracleStartingTimestamp);
-        checkOptimismMintableERC20Factory(prox);
-        checkL1ERC721Bridge(prox);
-        checkOptimismPortal(prox, cfg);
-        checkProtocolVersions(prox, cfg);
+        checkSystemConfig(_prox, _cfg, true);
+        checkL1CrossDomainMessenger(_prox, _vm);
+        checkL1StandardBridge(_prox);
+        checkL2OutputOracle(_prox, _cfg, _l2OutputOracleStartingTimestamp, _l2OutputOracleStartingBlockNumber);
+        checkOptimismMintableERC20Factory(_prox);
+        checkL1ERC721Bridge(_prox);
+        checkOptimismPortal(_prox, _cfg, false);
+        checkProtocolVersions(_prox, _cfg, true);
```

Comment:
At least for the three functions that accept a boolean, it would be helpful to use the kwargs call syntax, since in the portal it maps to paused, but in `checkSystemConfig` and `checkProtocolVersions` it maps to `bool _proxy`.